src/utils/is.ts

```diff
@@ -0,0 +1,32 @@
+import type { Contact, Owner } from "../services/hubspot/types";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isString = (value: any): value is string => {
+    return typeof value === "string";
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isNumber = (value: any): value is number => {
+    return typeof value === "number" && !isNaN(value);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isPrimitive = (value: any): value is string | number => {
```

Comment:
Not sure that just numbers and strings a primitives, are `null` and `boolean` ones too? It's probably best to no abstract this to helper functions just write what you need at the tim.